pennylane/io/qasm_interpreter.py

```diff
@@ -46,6 +48,9 @@
     "U1": ops.U1,
     "U2": ops.U2,
     "U3": ops.U3,
+    "CU": lambda theta, phi, delta, gamma, wires: ops.ctrl(
+        ops.GlobalPhase(gamma) @ ops.U3(theta, phi, delta, wires[1]), wires[0]
+    ),
```

Comment:
```suggestion
    "CU": lambda theta, phi, delta, gamma, wires: ops.PhaseShift(gamma) @ ops.ctrl(
        ops.U3(theta, phi, delta, wires[1]), wires[0]
    ),
```
the controlled global phase becomes a phase shift.